website/intro/getting-started/install.html.markdown

```diff
@@ -57,6 +57,17 @@ If you get an error that `terraform` could not be found, your `PATH` environment
 variable was not set up properly. Please go back and ensure that your `PATH`
 variable contains the directory where Terraform was installed.
 
+## Installing Terraform Providers
+
+Terraform uses a plugin based architecture to support the numerous infrastructure 
+and service providers available. As of Terraform version 0.10.0, each "Provider" is it's 
+own encapsulated binary distributed separately from Terraform itself. The core 
+Terraform binary downloaded and installed above will automatically download and 
+install any Provider binary needed for you, so you don't have to find and install 
+them yourself. A list of currently supported Providers can be found on 
+[our Providers page](/docs/providers/index.html).
```

Comment:
I agree that this seems like maybe too much for this stage in the guide.

Maybe instead of this section we can just add one more short paragraph to the `Installing Terraform` section above:

> Terraform providers are not included in the distribution zip file, but Terraform can install the necessary providers automatically when initializing a configuration. We'll deal with the automatic provider installation process in the next guide, so don't worry about providers for now.